packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import { Logger } from 'aws-amplify';
+import { changePassword, translate } from '@aws-amplify/ui';
+
+import { useAuth } from '../../../internal';
+import { View, Flex } from '../../../primitives';
+import {
+  DefaultCurrentPassword,
+  DefaultError,
+  DefaultNewPassword,
+  DefaultSubmitButton,
+} from './defaultComponents';
+import { ChangePasswordProps } from './types';
+import { FormValues } from '../types';
+
+const logger = new Logger('ChangePassword');
+
+const ChangePassword: React.ComponentType<ChangePasswordProps> = (props) => {
+  const { onSuccess, onError } = props;
+  const [error, setError] = React.useState<string>(null);
+  const [formValues, setFormValues] = React.useState<FormValues>({});
+
+  const { user, isLoading: isAuthUserLoading } = useAuth();
```

Comment:
Don't see a benefit in reassigning the `isLoading`?

Reply:
IMO I thought `isLoading` was somewhat ambiguous, because it sounds like the whole component is loading. What do you think?

Reply:
Since we return `null` as the component if `isLoading` is `true`, in a way we could say the the whole component is loading right?

Reply:
Fair, adjusting.

Reply:
https://github.com/aws-amplify/amplify-ui/blob/97936eec8e4fce7511c6c3c5deee9aed98b9984b/packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx#L26